Extension/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # C/C++ for Visual Studio Code Changelog
 
+### Version 1.23.2: December 5, 2024
+### Bug Fixes
+* Include clang-format timeout from 10 seconds to 30 seconds. [#10213](https://github.com/microsoft/vscode-cpptools/issues/10213)
+* Fix casing of path in include completion tooltip on Windows. [#12895](https://github.com/microsoft/vscode-cpptools/issues/12895)
+* Fix pattern matching of sections in `.editorConfig` files. [12933](https://github.com/microsoft/vscode-cpptools/issues/12933)
+* Fix handling of relative paths passed to cl.exe `/reference` argument. [#12944](https://github.com/microsoft/vscode-cpptools/issues/12944)
+* Fix a leak of compile command file watchers. [#12946](https://github.com/microsoft/vscode-cpptools/issues/12946)
+* Fix a compile commands fallback logic issue. [#12947](https://github.com/microsoft/vscode-cpptools/issues/12947)
+  * Thank you for the contribution. [@yiftahw](https://github.com/yiftahw) [#12948](https://github.com/microsoft/vscode-cpptools/pull/12948)
```

Comment:
Should 12946 have a thank you note too? Or should they be combined into 1 bullet point?

Reply:
I wasn't sure.  I guess both?

Reply:
Yeah, I think both should have a thank you or they should be combined into a single bullet point that references both issues.